risc0/risc0
# wip(benchmarks): react 19 / nextjs 15 testing

This PR includes these changes:
* update README logo and docs logo/favicon to new RISC Zero logo
* react compiler seems to break column visibility picker, so turned off for the time being -- can still safely migrate to react 19
* ppr now works properly
* updated favicon to be same as risczero.com website
* fixes the columns for the application benchmarks
* better error handling for all fetches
* stronger typings around versions
* tie applications benchmarks to version
* use correct URL for seo/meta
* properly redirect user if version doesn't exist (example https://benchmarks.risczero.com/release-0.11) 
* 3 mins cache for everything
* rename some files to be more consistent
* bump deps to latest
* remove dead code
* add meta keywords for bette SEO
* bit better sitemap generation

in the future, I'd like to improve a bit the automation, stuff like having automatic integration for newer releases, etc. will see how we can improve.
